documents/api/viewsets.py

```diff
@@ -30,10 +31,70 @@
     CreateAttachmentSerializer,
     DocumentSerializer,
 )
+from ..serializers.document import DocumentMetadataSerializer
 from ..serializers.status_history import StatusHistorySerializer
-from .docs import attachment_viewset_docs, document_viewset_docs
-from .filtersets import DocumentFilterSet
-from .querysets import get_attachment_queryset, get_document_queryset
+from .docs import (
+    attachment_viewset_docs,
+    document_metadata_viewset_docs,
+    document_viewset_docs,
+)
+from .filtersets import DocumentFilterSet, DocumentMetadataFilterSet
+from .querysets import (
+    get_attachment_queryset,
+    get_document_metadata_queryset,
+    get_document_queryset,
+)
+
+
+@extend_schema_view(**document_metadata_viewset_docs)
+class DocumentMetadataViewSet(AuditLoggingModelViewSet):
+    serializer_class = DocumentMetadataSerializer
+    queryset = Document.objects.none()
+    filter_backends = [DjangoFilterBackend]
+    lookup_field = "user__uuid"
+    pagination_class = PageNumberPagination
+    filterset_class = DocumentMetadataFilterSet
+
+    def get_queryset(self):
+        super().get_queryset()
```

Comment:
Is this obsolete row? Or what this is supposed to do?

Reply:
It's useless, well spotted. Thank you! I'll fix it.